vendor-bin/php-codesniffer/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "require": {
+    "php": "^7.3 || ^8.0",
+    "squizlabs/php_codesniffer": "^3.6",
```

Comment:
Could you also update the package versions in all composer.json files within vendor-bin?